src/zc/relation/README.txt

```diff
@@ -109,6 +111,12 @@ name as the tokens.
     ...         return '<Employee instance "' + self.name + '">'
     ...     def __lt__(self, other):
     ...         return self.name < other.name
+    ...     def __eq__(self, other):
+    ...         return False
```

Comment:
Maybe stick with `self is other` like in the other two cases?

Reply:
Fixed, thanks!